ios/FluentUI/Vnext/List/ListHeaderFooter.swift

```diff
@@ -6,23 +6,24 @@
 import UIKit
 import SwiftUI
 
-struct Header: View {
-    @Environment(\.theme) var theme: FluentUIStyle
-    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
-    @ObservedObject var tokens: MSFHeaderFooterTokens
-    @ObservedObject var state: MSFListSectionStateImpl
-
+struct Header: View, TokenizedControlInternal {
     init(state: MSFListSectionStateImpl) {
         self.state = state
-        self.tokens = state.headerTokens
     }
 
     var body: some View {
+        let backgroundColor: Color = {
+            guard let stateBackgroundColor = state.backgroundColor else {
+                return Color(dynamicColor: tokens.backgroundColor)
+            }
+            return Color(stateBackgroundColor)
```

Comment:
Do we still need the background color in the state, or could we just move it to the tokens and get custom colors from custom tokens?

Reply:
Resolved offline, will be removing backgroundColor in a future PR and it'll be controlled by custom tokens